README.md

```diff
@@ -29,6 +29,10 @@ You might want to start this app on different port, just add port number as para
 $ java -jar target/nerodesk-jar-with-dependencies.jar 9000
 ```
 
+Non-functional requirement is:
```

Comment:
There's only one bullet on this list. We're better off here having a sentence or paragraph saying to the effect that, "Nerodesk requires Java 8 (or higher)". Also, it would be good if we reiterate this fact in the "How to contribute" section below (after it is stated that Maven 3.2+ is required).